lib/util.ex

```diff
@@ -67,27 +73,33 @@ defmodule Util do
       Authorization: "Bearer " <> auth
     ]
 
-    {:ok, resp} = HTTPoison.post("https://api.start.gg/gql/alpha", JSON.encode!(body), headers)
-
-    resp.body
-    |> JSON.decode!()
-    |> Map.get("data")
-    |> Map.get("tournaments")
-    |> Map.get("nodes")
-    # Finesse data into our own non-start.gg format.
-    |> Enum.map(fn sgg_data ->
-      %{
-        "external_id" => "smashgg-" <> Integer.to_string(sgg_data["id"]),
-        "name" => sgg_data["name"],
-        "location" => %{
-          "lat" => sgg_data["lat"],
-          "lng" => sgg_data["lng"]
-        },
-        "start_time" => sgg_data["startAt"],
-        "end_time" => sgg_data["endAt"],
-        "updated_at" => sgg_data["updatedAt"],
-        "url" => "https://start.gg/" <> (sgg_data["slug"])
-      }
-    end)
+    case HTTPoison.post("https://api.start.gg/gql/alpha", JSON.encode!(body), headers) do
+      {:ok, resp} ->
+        case JSON.decode(resp.body) do
+          {:ok, response = %{"success" => false}} ->
+            IO.puts(Map.get(response, "message"))
```

Comment:
Let's also call `System.stop(1)` here and anywhere else that we encounter an error that can't be handled. I want to make sure we stop fully, and whatever automation is calling this script knows something went wrong.

(I wouldn't want the script to keep running and overwrite the tournaments file with bad data, for example.)